ibis/backends/oracle/__init__.py

```diff
@@ -380,6 +408,8 @@ def _register_in_memory_table(self, op: ops.InMemoryTable) -> None:
                 for row in data:
                     cur.execute(insert_stmt, row)
 
+        atexit.register(self._clean_up_tmp_table, name)
```

Comment:
Does oracle not drop temporary tables when the connection closes?

Reply:
There's some weirdness here and I think I don't have a good enough understanding at the moment of Oracle's notions of connections vs. sessions vs a few other terms that get thrown around in the docs.